frontend/packages/kubevirt-plugin/src/topology/TopologyVmPanel.tsx

```diff
@@ -45,10 +45,10 @@ export const ConnectedTopologyVmPanel: React.FC<TopologyVmPanelProps> = ({
   const vmData = vm.getData();
   const vmObj = vmData.resources.obj;
   const { namespace } = vmObj.metadata;
+  const actions = vmActions(vm.getData());
   const detailsComponent = () => <TopologyVmDetailsPanel vm={vm} />;
   const resourcesComponent = () => <TopologyVmResourcesPanel vmNode={vm} />;
```

Comment:
let's remove these inefficient wrappers and pass the components directly (now possible with combination with observer)

Reply:
These component needs to be in this form because of the `SimpleTabNav` component

Reply:
Have you tried? Just from looking at the code it should be possible to pass the components directly and unify/fix their props